lib/config/options/index.ts

```diff
@@ -229,6 +229,16 @@ const options: RenovateOptions[] = [
     default: 'disabled',
     experimental: true,
   },
+  {
+    name: 'repositoryCacheType',
+    description:
+      'Set the type of renovate repository cache if repositoryCache is not disabled.',
+    globalOnly: true,
+    type: 'string',
+    stage: 'repository',
+    default: 'local',
+    experimental: true,
```

Comment:
add allowed values even if it's only one

Reply:
got it, now i understand what you meant 😅